src/missed_spans.rs

```diff
@@ -46,6 +46,13 @@ impl<'a> FmtVisitor<'a> {
     // TODO these format_missing methods are ugly. Refactor and add unit tests
     // for the central whitespace stripping loop.
     pub fn format_missing(&mut self, end: BytePos) {
+        // HACK
```

Comment:
Could you copy the commit message here please so that we know more about the HACK in the code?